assets/css/style.css

```diff
@@ -373,6 +371,10 @@ nav ul li a:active {
 }
 
 @media (max-width: 950px) {
+  .introduction {
```

Comment:
Even in the mobile view
![image](https://github.com/rbjks/rbjks.github.io/assets/43912470/86ccde23-f7dd-46c0-a695-93d412b80730)

I'm not happy about how close the text is to the margins and the image above. Perhaps we could add some space around the text here?